tools/scripts/prplMesh-platform-init.sh

```diff
@@ -24,8 +24,10 @@ start_function() {
     for netif in /sys/class/mac80211_hwsim/*/net/*; do
         nmcli d set ${netif##*/} managed no
     done
-    sleep 1
-    run $HOSTAPD -P /var/run/hostapd.pid -B $HOSTAPD_CONF
```

Comment:
Do we really want to force the use of containers always? maybe add this as an option instead?

Reply:
I've added a third radio and start hostapd both inside and outside of the container.